package.json

```diff
@@ -51,7 +51,7 @@
     "chain-function": "^1.0.1",
     "classnames": "^2.2.6",
     "detect-browser": "^5.1.0",
-    "fundamental-styles": "0.11.3-rc.7",
+    "fundamental-styles": "0.11.3-rc.8",
```

Comment:
I don't see any changes that would affect us here: https://github.com/SAP/fundamental-styles/compare/v0.11.3-rc.7...v0.11.3-rc.8